core/server/server.js

```diff
@@ -482,6 +487,7 @@ class Server {
   static resetGlobalState() {
     // This state should be migrated to instance state. When possible, do not add new
     // global state.
+    clearRequestCache()
```

Comment:
This would be problematic but will go away after a rebase :+1:

Reply:
@danielle since we've been chatting about this offline, I'll mention it's fine to use rebase here, or to merge master into this branch. Then add another commit, if necessary, to reset this file.